src/view/components/splash/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './index.scss';
+
+interface Props {
+    progress: number;
+}
+
+const Splash: React.FC<Props> = ({ progress }) => {
+    return (
+        <div className="container" id="splash">
+            <img className="logo" src="/logo.png" alt="Logo" />
+            <div className="progress-bar">
+                <div className="progress" style={{ width: `${progress}%` }} />
+            </div>
+        </div>

```

Comment:
Please use nested class/id names
e.g. `splash-wrapper` instead of `container`, `splash-logo` instead of `logo`, `splash-progress-bar`, `splash-progress-bar-progress`